From 5459c34bee0624167e30fd0741b913e68090a144 Mon Sep 17 00:00:00 2001
Message-ID: <5459c34bee0624167e30fd0741b913e68090a144.1705743628.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 17 Jan 2024 00:16:28 +0000
Subject: [PATCH 27/38] gnu: go-github-com-operatorfoundation-ed25519: Move to
 (gnu packages golang-crypto).

* gnu/packages/golang.scm (go-github-com-operatorfoundation-ed25519):
Move from here ...
* gnu/packages/golang-crypto.scm: ... to here.

Change-Id: Ia6a28bb8ad138b921e386f3b4d2dee65b43806d1
---
 gnu/packages/golang-crypto.scm | 32 ++++++++++++++++++++++++++++++++
 gnu/packages/golang.scm        | 31 -------------------------------
 2 files changed, 32 insertions(+), 31 deletions(-)

diff --git a/gnu/packages/golang-crypto.scm b/gnu/packages/golang-crypto.scm
index 7216f63014..2b2e9d877f 100644
--- a/gnu/packages/golang-crypto.scm
+++ b/gnu/packages/golang-crypto.scm
@@ -4,6 +4,7 @@
 ;;; Copyright © 2019 Vagrant Cascadian <vagrant@debian.org>
 ;;; Copyright © 2019, 2020 Leo Famulari <leo@famulari.name>
 ;;; Copyright © 2021 Arun Isaac <arunisaac@systemreboot.net>
+;;; Copyright © 2021 LibreMiami <packaging-guix@libremiami.org>
 ;;; Copyright © 2021 Raghav Gururajan <rg@raghavgururajan.name>
 ;;; Copyright © 2022 (unmatched-parenthesis <paren@disroot.org>
 ;;; Copyright © 2022, 2023 Nicolas Graves <ngraves@ngraves.fr>
@@ -521,6 +522,37 @@ (define-public go-github-com-quic-go-qtls-go1-20
 QUIC.  For Go 1.20.")
     (license license:expat)))
 
+(define-public go-github-com-operatorfoundation-ed25519
+  (let ((commit "b22b4bd3ddef042eec45f3ee135cd40281fde2b4")
+        (revision "0"))
+    (package
+      (name "go-github-com-operatorfoundation-ed25519")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/OperatorFoundation/ed25519")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32 "0xrzqrjlghkgd1cy5rj4khryn4f59vas2vzrxc6d8jpj5ijf3xkv"))))
+      (build-system go-build-system)
+      (arguments
+       `(#:import-path "github.com/OperatorFoundation/ed25519"
+         #:phases
+         (modify-phases %standard-phases
+           (add-before 'install 'remove-test-data
+             (lambda* (#:key import-path #:allow-other-keys)
+               (delete-file-recursively
+                (string-append "src/" import-path "/testdata"))
+               #t)))))
+      (home-page "https://github.com/OperatorFoundation/ed25519")
+      (synopsis "Ed25519 for go")
+      (description "Package ed25519 implements the Ed25519 signature
+algorithm.")
+      (license license:bsd-3))))
+
 (define-public go-github-com-protonmail-go-crypto
   (package
     (name "go-github-com-protonmail-go-crypto")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index f77185b87b..581594a1e0 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -9850,37 +9850,6 @@ (define-public go-github-com-songmu-gitconfig
     (description "@{gitconfig} is a package to get configuration values from gitconfig.")
     (license license:expat)))
 
-(define-public go-github-com-operatorfoundation-ed25519
-  (let ((commit "b22b4bd3ddef042eec45f3ee135cd40281fde2b4")
-        (revision "0"))
-    (package
-      (name "go-github-com-operatorfoundation-ed25519")
-      (version (git-version "0.0.0" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                       (url "https://github.com/OperatorFoundation/ed25519")
-                       (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "0xrzqrjlghkgd1cy5rj4khryn4f59vas2vzrxc6d8jpj5ijf3xkv"))))
-      (build-system go-build-system)
-      (arguments
-       `(#:import-path "github.com/OperatorFoundation/ed25519"
-         #:phases
-         (modify-phases %standard-phases
-           (add-before 'install 'remove-test-data
-             (lambda* (#:key import-path #:allow-other-keys)
-               (delete-file-recursively
-                 (string-append "src/" import-path "/testdata"))
-               #t)))))
-      (home-page "https://github.com/OperatorFoundation/ed25519")
-      (synopsis "Ed25519 for go")
-      (description "Package ed25519 implements the Ed25519 signature
-algorithm.")
-      (license license:bsd-3))))
-
 (define-public go-github-com-akosmarton-papipes
   (let ((commit "3c63b4919c769c9c2b2d07e69a98abb0eb47fe64")
         (revision "0"))
-- 
2.41.0

